src/content/api/assets.mdx

```diff
@@ -160,6 +160,53 @@ Tokens have the following attributes along with the base asset fields.
 
 ---
 
+## Ledger Model <Badge type="experimental"/>
+
+Ledgers are a container for tokens. Ledgers contain a balance of the number of tokens the account has from a specific collection. They can be spent and sent in arbitrary amounts.
```

Comment:
Suggestion: `A ledger is a single asset that maintains a balance of multiple tokens belonging to an account. All tokens in a ledger come from a specific collection. Multiple tokens can be spent or sent at once from the ledger.`